acceptance/invoke/pack.go

```diff
@@ -195,21 +195,14 @@ func (i *PackInvoker) Supports(command string) bool {
 
 	var cmdParts = []string{"help"}
 
-	var search string
-	switch len(parts) {
-	case 1:
-		search = parts[0]
-	case 2:
-		cmdParts = append(cmdParts, parts[0])
-		search = parts[1]
-	default:
-		return false
-	}
+	last := len(parts) - 1
+	cmdParts = append(cmdParts, parts[:last]...)
+	search := parts[last]
 
 	output, err := i.baseCmd(cmdParts...).CombinedOutput()
 	i.assert.Nil(err)
 
-	return strings.Contains(string(output), search)
+	return strings.Contains(string(output), search) && !strings.Contains(string(output), "Unknown help topic")
```

Comment:
Since many of the command descriptions include "builder", it was falsely returning true, even though the initial line said "Unknown help topic". This ensures that we don't get false positives.